README.md

```diff
@@ -1,15 +1,15 @@
 # virtual-controlplane
 
+![Gardener on Metal Logo](docs/assets/logo.png)
+
 This project contains a collection of helm charts to deploy a virtual/nodeless Kubernetes API server.
 
-The API server can be deployed using various storage backends
+## Start using or developing the virual-controlplane locally
+
+Please see our [documentation](https://onmetal.github.io/virtual-controlplane/) or the content in the `/docs` folder for more details.
```

Comment:
For more details please check [documentation](https://onmetal.github.io/virtual-controlplane/) or the content in the `/docs` folder